src/App.js

```diff
@@ -9,7 +10,12 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1>React Article</h1>
+    {Article({
+      ...article,
+    })}
```

Comment:
```suggestion
    <Article { ...article } />
```
Компоненты вызываются вот так